sbin/build.sh

```diff
@@ -988,6 +988,28 @@ createNoticeFile() {
   fi
 }
 
+# Adds a script for easy JRE generation using jlink
+addCreateJREScript() {
```

Comment:
Let's not call it a JRE as that's an official designation.  I'd change it to Runtime or something like that